torch/csrc/jit/mobile/import.cpp

```diff
@@ -294,12 +295,6 @@ void BytecodeDeserializer::parseFunctionSchema(
   }
 }
 
-void BytecodeDeserializer::init_upgrader(mobile::Function* function) {
```

Comment:
Ok looks like we are trying to re-use this code...is there a better place to put re-use code between `import.cpp` and `flatbuffer_load.cpp`?